service/src/main/java/tds/assessment/services/impl/AssessmentServiceImpl.java

```diff
@@ -22,19 +25,17 @@
 @Service
 class AssessmentServiceImpl implements AssessmentService {
     private final AssessmentQueryRepository assessmentQueryRepository;
-    private final ItemQueryRepository itemQueryRepository;
-    private final FormQueryRepository formQueryRepository;
+    private final AssessmentAssembler assessmentAssembler;
     private final StrandQueryRepository strandQueryRepository;
 
     @Autowired
     public AssessmentServiceImpl(AssessmentQueryRepository assessmentQueryRepository,
                                  ItemQueryRepository itemQueryRepository,
                                  FormQueryRepository formQueryRepository,
                                  StrandQueryRepository strandQueryRepository) {
-        this.formQueryRepository = formQueryRepository;
         this.assessmentQueryRepository = assessmentQueryRepository;
-        this.itemQueryRepository = itemQueryRepository;
         this.strandQueryRepository = strandQueryRepository;
+        this.assessmentAssembler = new AssessmentAssembler(itemQueryRepository, formQueryRepository);
```

Comment:
Agreed with Tom. I think the idea is that AssessmentService should fetch all data (using the various repositories), and the Assembler is responsible for constructing the actual Assessment.